### PR TITLE
Support three-finger gesture to change pitch

### PR DESCRIPTION
**Pull request by Pessimistress**
For #5089

#### Change List
- Bump mjolnir.js dependency
- 3-finger pan handling in Controller
- 3-finger pan to change pitch in MapController
- Unit tests

### CODE REVIEW

File: docs/api-reference/core/controller.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The base Controller class supports the following options:
 * `dragRotate` (`Boolean`) - enable rotating with pointer drag. Default `true`
 * `doubleClickZoom` (`Boolean`) - enable zooming with double click. Default `true`
 * `touchZoom` (`Boolean`) - enable zooming with multi-touch. Default `true`
-* `touchRotate` (`Boolean`) - enable rotating with multi-touch. Default `false`
+* `touchRotate` (`Boolean`) - enable rotating with multi-touch. Use two-finger rotating gesture for horizontal and three-finger swiping gesture for vertical rotation. Default `false`
```
Comment (ibgreen) [inline docs/api-reference/core/controller.md:17]: Q: Useful to enable this separately `verticalRotate` etc?

File: modules/core/src/controllers/controller.js [hunk 2]
```diff
@@ -99,6 +101,12 @@ export default class Controller {
         return this._onPinch(event);
       case 'pinchend':
         return this._onPinchEnd(event);
+      case 'tripanstart':
+        return this._onTriplePanStart(event);
```
Comment (ibgreen) [inline modules/core/src/controllers/controller.js:105]: Total nit: These per-event methods are nice but as the set of events grows this generates a lot of non-minifiable method names (compared to e.g. `onEvent` `switch(event.type)` statement).

**Review by ibgreen — APPROVED** [review]
Can't wait to try it out!

File: docs/api-reference/core/controller.md (hunk 1 above)
Comment (Pessimistress) [reply docs/api-reference/core/controller.md:17]: The `dragRotate` switch on desktop controls both pitch and bearing.

**Pull request merged**